apis/scheduling/v1alpha1/types.go

```diff
@@ -47,13 +47,15 @@ type ElasticQuota struct {
 // ElasticQuotaSpec defines the Min and Max for Quota.
 type ElasticQuotaSpec struct {
 	// Min is the set of desired guaranteed limits for each named resource.
+	// +kubebuilder:default={cpu:0,memory:0,ephemeral-storage:0}
 	// +optional
```

Comment:
Let's remove `+optional` as well to keep consistency (given `omitempty` was deleted).

Reply:
Thank you for your feedback.
I noticed that if I remove the optional tag, I am unable to apply the following Yaml, which is not what I intended.
Also I realized that I don't need to remove the `omitempty` tag. Therefore, I have decided to keep both `+optional` and `omitempty` tags.
```
apiVersion: scheduling.sigs.k8s.io/v1alpha1
kind: ElasticQuota
metadata:
  name: test
  namespace: test
spec:
  min:
    cpu: 20
    memory: 40Gi
    nvidia.com/gpu: 2
```